### PR TITLE
Improve crew page: display goals, refine UI

### PR DESCRIPTION
**Pull request by igorkofman**
https://www.dropbox.com/s/94fucvqyvgf2s4x/Screenshot%202019-06-24%2017.57.06.png?dl=0

### CODE REVIEW

**Comment by igorkofman** [comment]
Security (restricting mission cancellation to captains) to be added in a future change

**Comment by igorkofman** [comment]
Feedback from Sasha last night (to be addressed in the future):
- what the primary title at the top of the page is still not clear enough
- wants to be able to post and see photos / progress updates during the mission
   (I think the MVP version of this is the Mission start email thread having everyone CC'd on it with an invitation to post updates)
- wants to be able to read more background about the mission&goal.  I think an MVP would be a "Mission Briefing" section in the mission box that would contain links to off-site articles associated with the current Mission

Additionally I'd love to chat on Thursday about how to iterate on how we model missions/goals. I think what I have here is ok for MVP but we should come up with something better soon after.

File: spaceship/views.py [hunk 1]
```diff
@@ -121,28 +122,53 @@ def confirm_token(token, expiration=3600):
...
+    return redirect(url_for('home'))
+  if mission.is_deleted:
+    flash({'msg':f'Mission was already cancelled.', 'level':'danger'})
+    return redirect(url_for('home'))
+
+  try:
+    mission.deleted_at = pendulum.now('UTC')
+    mission.save()
+  except DatabaseError:
+    flash({'msg':f'Database error', 'level':'danger'})
```
Comment (igor47) [inline spaceship/views.py:140]: should probably log something when this happens

File: spaceship/views.py [hunk 2]
```diff
@@ -121,28 +122,53 @@ def confirm_token(token, expiration=3600):
         return False
     return email
 
+@app.route('/mission/<mission_id>/cancel', methods=['POST'])
+@login_required
+def cancel_mission(mission_id):
+  mission = Mission.query.filter(Mission.id == mission_id).first()
```
Comment (igor47) [inline spaceship/views.py:128]:
```python
if mission.team.captain != current_user:
  flash()
  return redirect()
```

**Review by igor47 — APPROVED** [review]
looks great!

**Pull request merged**